speed-transition.lua

```diff
@@ -52,13 +53,21 @@ function speed_transition(_, sub)
          mark = speedup_zone_begin
          speedup_zone_end = mark+nextsub
          if shouldspeedup then
-            normalspeed = mp.get_property("speed")
-            if mp.get_property_native("video-sync") == "audio" then
-               mp.set_property("video-sync", "desync")
+            if skipmode and (nextsub-leadin>=leadin or nextsub==0) and mp.get_property("pause") == "no" then
+               if nextsub>set_timeout() or nextsub==0 then
```

Comment:
`nextsub>set_timeout()` should be `nextsub>set_timeout()-leadin` since else it can skipt to far

Reply:
will change